tools/tools.yml

```diff
@@ -241,6 +241,12 @@
       badges: pypi, rtd
       builtons: [opengl]
 
+    - repo: datoviz/datoviz
+      site: https://datoviz.org
+      conda_channel: mark.harfouche
```

Comment:
Is this an official distribution channel for datoviz? I don't see anyone by that name in the datoviz contributors list. If it is just a single contribution from a user, I don't think it's appropriate to list here.

Reply:
Sorry, I haven't checked whether it is an official channel. Now I have found the same user with username _hmaarrfk_ in the list of datoviz contributors.

Reply:
Ok, if that's the same person, it's good enough for me! Thanks.